pkg/controllers/allocation/controller.go

```diff
@@ -176,7 +195,12 @@ func (c *Controller) podToProvisioner(o client.Object) (requests []reconcile.Req
 	ctx := context.Background()
 	provisioner, err := c.getProvisionerFor(ctx, pod)
 	if err != nil {
-		zap.S().Errorf("Retrieving provisioner, %s", err.Error())
+		if errors.IsNotFound(err) {
+			// Queue and batch a reconcile request for a non-existent, empty provisioner
+			// This will reduce the number of repeated error messages about a provisioner not existing
+			c.Batcher.Add(&v1alpha3.Provisioner{})
```

Comment:
This feels like a nasty hack, but I don't have a better suggestion.

Reply:
:)